manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: dbeaver.dbeaver
+PackageVersion: 21.1.2
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
+  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+    Custom: /currentuser
```

```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
  InstallerSwitches:
    Custom: /ALLUSERS
```
Only reviewer who will correct this 🙄 Needed for compatibility, at least when the default installation mode is for all users